_posts/it/2021-11-09-hacktoberfest-risultati.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "Hacktoberfest 2021: i risultati"
+subtitle: Developers Italia ha partecipato all’ottava edizione dell’evento internazionale sull’open source
+date: 2021-11-09 17:00:00 +0200
+tags: developers api opensource
```

Comment:
```suggestion
tags: hacktoberfest developers api opensource
```
There's also the `hacktoberfest` tag https://developers.italia.it/it/news/hacktoberfest